front/src/applications/operationalStudies/types.ts

```diff
@@ -156,10 +157,10 @@ export type ElectrificationVoltage = {
 };
 
 export type SimulationResultsData = {
-  selectedTrainSchedule?: TrainScheduleResult;
-  selectedTrainRollingStock?: RollingStockWithLiveries;
-  selectedTrainPowerRestrictions: LayerData<PowerRestrictionValues>[];
-  trainSimulation?: SimulationResponseSuccess;
+  selectedTimetableItem?: TrainScheduleResult | PacedTrainResponse;
```

Comment:
Why not a timetableItem here ? It would be easier to manipulate (for the ids)
```suggestion
  selectedTimetableItem?: TimetableItem;
```

Reply:
Done (many renaming in files be prepared 😅)